src/continuation/continuation_afam.jl

```diff
@@ -0,0 +1,107 @@
+"""
+    AttractorsContinueAndMatch(mapper, matcher [, seeding])
+
+A continuation method for [`continuation`](@ref).
+`mapper` is any subtype of [`AttractorMapper`](@ref) which implements
+[`extract_attractors`](@ref), while `matcher` is any subtype of [`SSSetMatcher`](@ref).
+
+## Description
+
+This global continuation method has a simple basis: it continues attractors by
+"seeding" initial conditions from previously found attractors.
+
+At the first parameter slice of the continuation process, attractors and their fractions
+are found using the given `mapper`.
+At each subsequent parameter slice,
+new attractors are found by seeding initial conditions from the previously found
+attractors and then running these initial conditions through the `mapper`.
+Seeding initial conditions close to previous attractors increases the probability
+that if an attractor continues to exist in the new parameter, it is found.
+Seeding is controlled by the `seeding` input. It is a function that given
+a state space set (an attractor) it returns an _iterator_ of initial conditions
+sampled from the attractor. This generates the seeds.
+
+After the special seeded initial conditions are mapped to attractors,
+attractor basin fractions are computed by sampling additional initial conditions
+using the provided `ics` in [`continuation`](@ref)).
+I.e., exactly as in [`basins_fractions`](@ref).
+Naturally, during this step new attractors may be found, besides those found
+using the "seeding from previous attractors".
+Once the basins fractions are computed,
+the parameter is incremented again and we perform the steps as before.
+
+This process continues for all parameter values. After all parameters are exhausted,
+the found attractors (and their fractions) are "matched" to the previous ones.
+This means: their _IDs are changed_, so that attractors that are "similar" to those at a
+previous parameter get assigned the same ID.
+Matching is done by the provided `matcher`.
+In code, matching is a rather trivial call to [`match_continuation!`](@ref)
+at the end of the `continuation` function.
+If you don't like the final matching output,
+you may use arbitrarily different `matcher` in [`match_continuation!`](@ref)
+without having to recompute the whole continuation.
+
+The matching algorithms are rather sophisticated,
+so how matching works is described in the docstrings of each `matcher`.
+"""
+struct AttractorsContinueAndMatch{A, M, S} <: AttractorsBasinsContinuation
```

Comment:
I like `AttractorsContinueAndMatch` , `AttractorsSeedAndMatch` do not refer to the continuation or matching process. In this new interface the seeding can be a random state space point generator also.

Reply:
I actually really want "seeding" to be part of the name because really that is what is unique about this continuation.